boilerplate/storybook/storybook.tsx

```diff
@@ -1,16 +1,19 @@
 import React from "react"
 import { getStorybookUI, configure } from "@storybook/react-native"
 import SplashScreen from "react-native-splash-screen"
+import { Reactotron } from "../src/services/reactotron"
 
 configure(() => {
   require("./storybook-registry")
 })
 
-const StorybookUI = getStorybookUI({ port: 9001, host: "localhost", onDeviceUI: true })
+const StorybookUI = getStorybookUI({ port: 9001, host: "", onDeviceUI: true })
```

Comment:
We probably want to leave this as it was, don't we?

Reply:
Actually I dont know. 

With "localhost" it was trying to  access to http://localhost/127.0.0.1/... 
So I removed that and it worked.

Reply:
Strange, I don't think I've experienced that behavior. What platform are you on?

Reply:
Ubuntu, Node v8.9.4, repo ignited yesterday.

Reply:
Chaps, check out #50. This might be the go.